doc/usecases/polyculture/suggest_groups.md

```diff
@@ -0,0 +1,31 @@
+# Use Case: Group suggestions
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user is provided with suggestions on groups of plants to use.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has opened the app.
+  - At least some of these layers are filled with data:
+    - Hydrology
+    - Terrain
+    - Shadows
+    - Soil
+    - Trees
+    - Winds
+    - Zones
+- **Main success scenario:**
+  - Groups of plants that are known to work well with each other and fit the environmental conditions are presented to the user.
```

Comment:
Please use enumeration or lists to avoid such sentence structures.

```suggestion
  - Groups of plants that are
    1. known to work well with each other and
    2. fit the environmental conditions
    are presented to the user.
```

Reply:
Sure.
https://github.com/ElektraInitiative/PermaplanT/pull/120/commits/fcf637c2c688b14ce483fb3091f7e570dd0f0db4